adaptorsensor/GUI.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package adaptorsensor;
+
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.GridLayout;
+import javax.swing.GroupLayout;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.border.TitledBorder;
+
+
+public class GUI extends JFrame{
+		public GUI(Adaptor a, Adaptor b, Adaptor c) {

```

Comment:
naming convention could be a little bit better. maybe "adaptor1","adaptor2","adaptor3" that way when referencing the variable you at least know that they are of the type Adaptor.

Reply:
or instead of calling them adaptors you could call them sensors